packages/react-composites/src/composites/CallWithChatComposite/EmbeddedPeoplePane.tsx

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { DefaultButton, FontIcon, PrimaryButton, Stack } from '@fluentui/react';
+import { ParticipantList } from '@internal/react-components';
+import copy from 'copy-to-clipboard';
+import React, { useMemo } from 'react';
+import { CallAdapter } from '../CallComposite';
+import { usePropsFor } from '../CallComposite/hooks/usePropsFor';
+import { ChatAdapter } from '../ChatComposite';
+import { AvatarPersonaDataCallback } from '../common/AvatarPersona';
+import { ParticipantListWithHeading } from '../common/ParticipantContainer';
+import { peoplePaneContainerTokens } from '../common/styles/ParticipantContainer.styles';
+import { useCallWithChatCompositeStrings } from './hooks/useCallWithChatCompositeStrings';
+import { MobilePane } from './MobilePane';
+import { SidePane } from './SidePane';
+import { copyLinkButtonStyles, linkIconStyles } from './styles/EmbeddedPeoplePane.styles';
+
+/**
+ * @private
+ */
+export const EmbeddedPeoplePane = (props: {
+  inviteLink?: string;
+  onClose: () => void;
+  hidden: boolean;
+  callAdapter: CallAdapter;
+  chatAdapter: ChatAdapter;
+  onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
+  onChatButtonClick: () => void;
+  onPeopleButtonClick: () => void;
+  mobileView?: boolean;
+}): JSX.Element => {
+  const { callAdapter, chatAdapter, inviteLink } = props;
+  const participantListDefaultProps = usePropsFor(ParticipantList);
+
+  const callWithChatStrings = useCallWithChatCompositeStrings();
+
+  const participantListProps = useMemo(() => {
+    const onRemoveParticipant = async (participantId: string): Promise<void> =>
+      removeParticipantFromCallWithChat(callAdapter, chatAdapter, participantId);
+    return {
+      ...participantListDefaultProps,
+      onRemoveParticipant
+    };
+  }, [participantListDefaultProps, callAdapter, chatAdapter]);
+
+  const participantList = (
+    <ParticipantListWithHeading
+      participantListProps={participantListProps}
+      onFetchAvatarPersonaData={props.onFetchAvatarPersonaData}
+      title={callWithChatStrings.peoplePaneSubTitle}
+    />
+  );
+
+  if (props.mobileView) {
+    return (
+      <MobilePane
+        hidden={props.hidden}
+        dataUiId={'call-with-chat-composite-people-pane'}
+        onClose={props.onClose}
+        activeTab="people"
+        onChatButtonClicked={props.onChatButtonClick}
+        onPeopleButtonClicked={props.onPeopleButtonClick}
+      >
+        <Stack verticalFill tokens={peoplePaneContainerTokens}>
+          {participantList}
+          {inviteLink && (
+            <PrimaryButton
+              onClick={() => copy(inviteLink)}
+              styles={copyLinkButtonStyles}
+              onRenderIcon={() => <FontIcon iconName="Link" style={linkIconStyles} />}
```

Comment:
This is using the old style link icon. We should use the new one https://iconcloud.design/search/filter/Fluent%20System%20Library/Fluent%20Regular/link/d0359d032-3821b375b

Will need imported from the react-icons package and made customizable by adding to the `DEFAULT_COMPOSITE_ICONS`

Probably best as a separate PR as call composite should be updated with this also